package.json

```diff
@@ -31,6 +31,7 @@
     "@angular/core": "^4.0.0",
     "@angular/forms": "^4.0.0",
     "@angular/http": "^4.0.0",
+    "@angular/router": "^4.0.0",
```

Comment:
This will still cause the generated code to reference symbols from the `@angular/router` package, though, meaning that the user will have to both install it _and_ have some non-zero payload from `@angular/router`.